doc/interns/t-ansokh/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+5/6/2020
```

Comment:
This is great!  But I forgot to add the time to the instructions.  Could you add time to this file?